documentation/1.2/tour/inheritance.md

```diff
@@ -299,11 +299,11 @@ an abstract superclass:
     }
 
 Ceylon requires us to annotate abstract classes `abstract`, just like Java. 
-This annotation specifies that a class cannot be instantiated, and can define 
-abstract members. Like Java, Ceylon also requires us to annotate "abstract" 
-members that don't specify an implementation. However, in this case, the 
-required annotation is `formal`. The reason for having two different 
-annotations, as we'll see 
+This annotation specifies that a class cannot be instantiated, and that it
+can define abstract members. Abstract members are members that don't specify
```

Comment:
I don’t think it’s right to call them abstract members, at least not without quotation marks (as in the original text). The point is that they’re *not* abstract – they’re formal, which is different.